README.md

```diff
@@ -28,5 +26,3 @@ control (start/stop/restart), logging, checkpointing and parallelization.
 For more information, please check http://mincong-h.github.io/gsoc-hsearch
```

Comment:
Could your please remove this line? The documentation is obsolete, so it's better to remove it too. I'll also disable the graphic branch `gh-pages`.